src/main/java/ch/njol/skript/bukkitutil/BukkitUnsafe.java

```diff
@@ -68,9 +65,32 @@ public class BukkitUnsafe {
 	@Nullable
 	private static Map<Integer,Material> idMappings;
 
+	/**
+	 * @deprecated You should use {@link BukkitUnsafe#getMaterialFromNamespacedId(String)}
```

Comment:
add why it's deprecated

Reply:
Bcs we renamed it

Reply:
that's not why though lol, you should say that this supports mod items

Reply:
sovdeeth done it  😂